### PR TITLE
G Cloud 9 contract variation

### PR DESCRIPTION
**Pull request by DilwoarH**
G Cloud 9 contract variation

As a supplier, I can review the proposed contract variation for the GCloud9 framework agreement and call-off contract, so that I understand what it includes and decide whether to accept it.

Ticket:
https://trello.com/c/ipSUk2Yo/104-add-final-copy-for-contract-variation-for-gcloud-9-and-update-feature-to-reflect-the-correct-dates

screenshot-localhost-2018 02 09-16-39-24

### CODE REVIEW

**Comment by risicle** [comment]
Wow vague vague parts of my memory being tickled by this PR.

File: VERSION.txt [hunk 1]
```diff
@@ -1 +1 @@
-10.3.0
+11.0.0
```
Comment (idavidmcdonald) [inline VERSION.txt]: I'd argue that this is not a breaking change and is still new functionality i.e it should become 10.4.0 (https://semver.org/)

File: frameworks/g-cloud-9/messages/contract_variation_1.yml [hunk 2]
```diff
@@ -0,0 +1,466 @@
+variation_description_not_in_place: |
+    The Crown Commercial Service (CCS) has made some minor changes to the framework agreement and call-off contract (reference RM1557ix) because of the new General Data Protection Regulation (GDPR).<br/><br/>
```
Comment (idavidmcdonald) [inline frameworks/g-cloud-9/messages/contract_variation_1.yml]: Comparing this to the g8 variation, on here we are including `br` tags whereas on the previous ones we didn't. I'm guessing they are nececssary?

File: VERSION.txt (hunk 1 above)
Comment (DilwoarH) [reply VERSION.txt]: fair point, il get that updated

File: frameworks/g-cloud-9/messages/contract_variation_1.yml [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+
+variation_agreement_label: I have the legal authority to accept these changes and agree that the company will be bound by them.
+
+confirmation_message: You have accepted the proposed changes. They will come into effect when CCS has countersigned them. We’ll let you know when this happens.
+
+framework_agreement_changes:
+    - 
+      clause: '8.57'
+      change: |
+            <span class="visuallyhidden">start insertion</span><ins>a)</ins>
```
Comment (idavidmcdonald) [inline frameworks/g-cloud-9/messages/contract_variation_1.yml]: Do we need an 'end insertion' accessibility span after the `</ins>`?

File: frameworks/g-cloud-9/messages/contract_variation_1.yml [hunk 4]
```diff
@@ -0,0 +1,466 @@
...
+    -
+      clause: 'Schedule 3 - Glossary and interpretations'
+      change: |
+          <span class="visuallyhidden">start insertion</span>
+            <ins>
+                <strong>Subprocessor</strong> <br/>
+                Any third party appointed to process Personal Data on behalf of the Supplier under this Framework Agreement.
+            </ins>
+          <span class="visuallyhidden">end insertion</span>
+call_off_contract_changes:
```
Comment (idavidmcdonald) [inline frameworks/g-cloud-9/messages/contract_variation_1.yml]: Would be good to have a blank line above this for better readability please

File: frameworks/g-cloud-9/messages/contract_variation_1.yml (hunk 2 above)
Comment (DilwoarH) [reply frameworks/g-cloud-9/messages/contract_variation_1.yml]: just a slight styling update

**Comment by DilwoarH** [comment]
@idavidmcdonald  - schema updated, tests are now passing 🍾 

Your comments addressed.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,12 @@
 
 Records breaking changes from major version bumps
 
+## 10.4.0
```
Comment (idavidmcdonald) [inline CHANGELOG.md]: No changelog record needing for a non breaking change. Your commit message should be the appropriate place for context I think.

**Comment by DilwoarH** [comment]
@idavidmcdonald updated

**Pull request merged**